e2e/testcases/helm_sync_test.go

```diff
@@ -907,7 +907,7 @@ func rootSyncForWordpressHelmChart(nt *nomostest.NT, valuesMutator func(map[stri
 	rs.Spec.Helm = &v1beta1.HelmRootSync{
 		Namespace: "wordpress",
 		HelmBase: v1beta1.HelmBase{
-			Repo:        "https://charts.bitnami.com/bitnami",
+			Repo:        "oci://registry-1.docker.io/bitnamicharts",
```

Comment:
Would it be feasible to switch this to a registry that we own so that we have more control over this?

Reply:
That's a good idea. To keep minimal changes to the test case, I'll just pull the workpress Helm chart from `oci://registry-1.docker.io/bitnamicharts` and push it to the public AR repo in the CI project.

Reply:
Yes, I'm writing a make target to pull from the public helm repo, and then push to our public AR repo.

There is a bummer that I hit an issue when trying to run the make target (aka. infra code) to push the same repo again.
This prevents us from re-building the test infra.

If there is no good way to bypass this issue, I would rather use the "real" public repo, `oci://registry-1.docker.io/bitnamicharts`.

Reply:
This PR is a good example of why to host our own infra instead of relying on an upstream repository - at any time the upstream maintainers could make a breaking change that breaks our CI overnight. Yes, the tradeoff is that we need to maintain some infra, but we are already hosting image registries for CI infra and it's a small amount of code to retag/push an image.

What is the error? Is it because the registry has immutable image tags? If so, just ensure the image we are re-pushing has a unique tag.

Reply:
I disabled the immutable image tag option. After deleting the helm charts and re-pushing with the same version, I got an error from the AR server: `Error: failed commit on ref "manifest-sha256:30edc537f73f3c981f575b2db0387d57f243b62b93224b16fe49f0c76a08cb43": cannot reuse body, request must be retried`.

Reply:
It might be some specific settings of the public AR repo of the CI project. I tried to re-push the same image with my own project, and it is working well.

Reply:
I updated the make target to check whether the helm chart version exists first to avoid re-pushing the same image.

Now we can use a public Helm chart from a AR repo that we have full control.